website/pages/badge.mdx

```diff
@@ -70,7 +70,7 @@ different style.
 
 <br />
 
-You can also change the size of the badge by passing `fontSize` prop.
+You can also change the size of the badge bypassing `fontSize` prop.
```

Comment:
Suggestion: This I think should remain as it was. 😊

Reply:
Yes, I see now that bypassing is wrong since it has a different meaning. Good catch!